db/migrate/20220706180933_createcomments.rb

```diff
@@ -0,0 +1,7 @@
+class Createcomments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.timestamps
+  end
+end
```

Comment:
- [ ] You are missing the association between the comment, the post table and the user table. It can be done here with `t.references :author, foreign_key: { to_table: 'users' }` # this will create author_id and a corresponding index  
`t.references :post, foreign_key: true`
Or you can also add column and index in separate commands